servant-server/servant-server.cabal

```diff
@@ -48,6 +48,7 @@ library
       , attoparsec         >= 0.12 && < 0.14
       , bytestring         >= 0.10 && < 0.11
       , containers         >= 0.5  && < 0.6
+      , http-api-data
```

Comment:
Do you want upper bounds for `servant-server` only or `servant` and `servant-client` as well?